drivers/input/input_ite_it8xxx2_kbd.c

```diff
@@ -62,7 +62,7 @@ static void it8xxx2_kbd_drive_column(const struct device *dev, int col)
 	const uint32_t kso_mask = BIT_MASK(common->col_size);
 	const uint8_t ksol_mask = kso_mask & 0xff;
 	const uint8_t ksoh1_mask = (kso_mask >> 8) & 0xff;
-	uint32_t kso_val;
+	uint32_t kso_val, key;
```

Comment:
I think key should be `unsigned int`, happens to be the same here but that's how `arch_irq_lock` is declared

Reply:
Done.